openbb_terminal/stocks/stocks_helper.py

```diff
@@ -672,6 +676,16 @@ def display_candle(
 
     data.name = f"{asset_type} {symbol}"
 
+    if ha:
+        data_ = heikin_ashi(data)
+        data["Open"] = data_["HA Open"]
+        data["High"] = data_["HA High"]
+        data["Low"] = data_["HA Low"]
+        data["Close"] = data_["HA Close"]
+
+    if ha:
+        data.name = f"{asset_type} {symbol} - Heikin Ashi Candles"
```

Comment:
The PR says:

Also touched is the default title, removing the arbitrary default label, "Stock".

This one has the asset type